src/main/java/dk/kb/image/util/KalturaUtil.java

```diff
@@ -36,7 +36,7 @@ public class KalturaUtil {
      * @return ThumbnailsDto. Has a default thumbnail, a sprite and list of time sliced thumbnails.
      * @throws IOException If the fileId is not found or internal server error with Kaltura.
      */
-    public static ThumbnailsDto getThumbnails(String fileId,Integer numberOfSlices, Integer width, Integer height) throws ServiceException {
+    public static ThumbnailsDto getThumbnails(String fileId,Integer numberOfSlices, Integer secondsStartSeek, Integer secondsEndSeek, Integer width, Integer height) throws ServiceException {
```

Comment:
Could you update the JavaDoc here to include information on what these new params does?

Reply:
javadoc added